runners/launch_docker-dev.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Usage: launch_docker-dev.sh
+# You can specify LOCAL_MNT and MJKEY_MNT ahead
+
+DOCKER_IMAGE="humancompatibleai/imitation:python-req"
+# Specify LOCAL_MNT if you want to mount a local directory to the docker container
+if [[ ${LOCAL_MNT} == "" ]]; then
+  LOCAL_MNT="${HOME}/imitation"
+fi
+
+# Pass your own mjkey.txt
+if [[ ${MJKEY_MNT} == "" ]]; then
+  MJKEY_MNT="${HOME}/mnt/mjkey.txt"
+fi
+
+# install imitation in developer mode
+CMD="pip install -e .[docs,parallel,test] gym[mujoco]"  # borrowed from ci/build_venv.sh
+
+# docker pull ${DOCKER_IMAGE}  # Uncomment this line to pull the image
+docker run -it --rm --init \
+       -v ${LOCAL_MNT}:/imitation \
+       -v ${MJKEY_MNT}:/root/.mujoco/mjkey.txt \
+       ${FLAGS} ${DOCKER_IMAGE} \
+       /bin/bash -c "${CMD} && bash"
```

Comment:
I haven't tested it, but won't this give us an additional bash process so that we have to exit twice to get out of the Docker container? I'm just using exec bash in my script and I don't think that has any drawbacks

Reply:
I tested and found the minimal version for me is `docker run -it --rm --init ${DOCKER_IMAGE} /bin/bash -c "${CMD} && bash"`. Deleting the first /bin/bash -c seems to results in XdummyDisplay errors. Deleting the second one works but will make the container stay in detached mode. I didn't find the case where I have to exit twice to get out of the container. Would be nice if you wanna test it.

Reply:
I meant `/bin/bash -c "${CMD} && exec bash"` but if you don't have to exit twice, then nvm, I apparently just don't understand Docker and shells as well as I'd thought :)

Reply:
@ejnnr I think you're right there's two shells, but when the innermost shell (launched by `&& bash`) exits, so will the outermost shell since it was run with `-c` which means "run this command then exit". `exec bash` has the effect of *replacing* the outermost shell with the innermost shell, so will save memory and I think is slightly better, although it's a negligible difference in practice.

Reply:
Thanks for pointing that out. I missed out `exec bash` you mentioned. I also tested it and it works. I'll make the change accordingly.

Reply:
It seems you're right. I missed out `exec bash`. It works, and I'll make changes accordingly